htdocs/takepos/admin/setup.php

```diff
@@ -241,6 +241,13 @@
 print $form->selectarray('TAKEPOS_EMAIL_TEMPLATE_INVOICE', $arrayofmessagename, $conf->global->TAKEPOS_EMAIL_TEMPLATE_INVOICE, 'None', 1, 0, '', 0, 0, 0, '', '', 1);
 print "</td></tr>\n";
 
+// Numbering module
+print '<tr class="oddeven"><td>';
+print $langs->trans("BillsNumberingModule");
+print '<td colspan="2">';
+$array = array(0=>$langs->trans("Default"), "terminal"=>$langs->trans("ByTerminal"), "mod_facture_mars"=>"mars", "mod_facture_mercure"=>"mercure", "mod_facture_terre"=>"terre");
```

Comment:
We should not a hard coded list as this may differs on each dolibarr installation.
You should get list of available numbering module in the same manner that it is done into the page to setup invoice.